src/bitcoind_client.rs

```diff
@@ -238,7 +238,20 @@ impl BroadcasterInterface for BitcoindClient {
 		let tx_serialized = serde_json::json!(encode::serialize_hex(tx));
 		tokio::spawn(async move {
 			let mut rpc = bitcoind_rpc_client.lock().await;
-			rpc.call_method::<RawTx>("sendrawtransaction", &vec![tx_serialized]).await.unwrap();
+			// This may error due to RL calling `broadcast_transaction` with the same transaction
+			// multiple times, but the error is safe to ignore.
+			match rpc.call_method::<RawTx>("sendrawtransaction", &vec![tx_serialized]).await {
+				Ok(_) => {}
+				Err(e) => {
+					let err_str = e.get_ref().unwrap().to_string();
+					if !err_str.contains("Transaction already in block chain")
+						&& !err_str.contains("Missing inputs")
```

Comment:
I only see "Missing inputs" logged in Core.

https://github.com/bitcoin/bitcoin/blob/ee9befe8b4bbd982f8c6eade556aea9b4d11a369/src/validation.cpp#L287

Should it be "Inputs missing or spent" instead?

https://github.com/bitcoin/bitcoin/blob/7fcf53f7b4524572d1d0c9a5fdc388e87eb02416/src/util/error.cpp#L17

As obtained by:

https://github.com/bitcoin/bitcoin/blob/adf78434100204f2eb799bbb96a28b734fb084f0/src/rpc/util.cpp#L353

Reply:
Hmm, I pulled "Missing inputs" out of the output against the node I was running against. It may be that core has changed the names of things since as that node is rather old.

Reply:
Ah, yeah blame log says as much:

https://github.com/bitcoin/bitcoin/commit/9e7b23b73387600d175aff8bd5e6624dd51f86e7

Reply:
Not sure if the error code returned can identify these without also examining the error message, but in the future we can consider parsing the JSON and examining it to be robust against such changes or support for translated messages.